topics/climate/tutorials/ocean-variables/tutorial.md

```diff
@@ -188,8 +420,10 @@ the "Project" Menu tab.
 {: .tip}
 
 ## Save Data
-{% include _includes/cyoa-choices.html option1="xview" option2="png" default="png"
+
+{% include _includes/cyoa-choices.html option1="xview" option2="png" default="png" disambiguation="savedata"
```

Comment:
@Marie59 I added `disambiguation="savedata"` to the second cyoa block to fix your problem. Whenever you have more than 1 cyoa in a tutorial, you need to provide this extra term so that the website knows which cyoa a chosen answer belongs to (see also [this contributing tutorial](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#choose-your-own-tutorial) for more info)